src/forms/FormActions.js

```diff
@@ -501,4 +509,23 @@ export const editAnswer = (edited, answer, formId) => {
   };
 };
 
+export const updateFilterBy = filterBy => ({
+  type: UPDATE_FILTER_BY,
+  value: filterBy
+});
+
+export const updateOrder = order => ({
+  type: UPDATE_ORDER,
+  value: order
+});
+
+export const updateSearch = search => ({
+  type: UPDATE_SEARCH,
+  value: search
+});
+
+export const cleanSubmissionFilters = () => ({
+  type: CLEAN_SUBMISSION_FILTERS
+});
+
```

Comment:
Line 530: this file keeps getting bigger and bigger, not for this PR, but maybe we could start splitting some of this stuff.